### PR TITLE
Add v1 & v2 namespace tests

### PR DESCRIPTION
**Pull request by jlevin**

### CODE REVIEW

File: vg/_helpers.py [hunk 1]
```diff
@@ -67,3 +67,9 @@ def _check_value_any(a, *shapes, name=None):
         raise ValueError(f"{preamble} with shape {shape_choices}; got None")
     else:
         raise ValueError(f"{preamble} with shape {shape_choices}; got {a.shape}")
+
+
+# adapted from https://stackoverflow.com/a/31465848
+def get_imported_names(module):
```
Comment (paulmelnikow) [inline vg/_helpers.py:73]: Is the purpose of this function to work before and after the addition of `__all__` to one of these modules?

Comment (jlevin) [reply vg/_helpers.py:73]: yes, that was my thinking.

Comment (paulmelnikow) [reply vg/_helpers.py:73]: I think it’s probably not worth it, I would just inline the bit of the if that you need in each test, and swap it out when you add `__all__` in the next PR.

Comment (paulmelnikow) [reply vg/_helpers.py:73]: Alternatively it should go in a test helpers module which should get excluded at build time so it doesn’t get shipped with the library.

Comment (jlevin) [reply vg/_helpers.py]: done and done

**Review by paulmelnikow — APPROVED** [review]

**Pull request merged**